src/ArduinoIoTCloudTCP.cpp

```diff
@@ -641,9 +593,19 @@ void ArduinoIoTCloudTCP::handle_OTARequest() {
 
 ArduinoIoTCloudTCP::State ArduinoIoTCloudTCP::handle_Disconnect()
 {
-  DEBUG_ERROR("ArduinoIoTCloudTCP::%s MQTT client connection lost", __FUNCTION__);
-  _mqttClient.stop();
+  if (!_mqttClient.connected()) {
+    DEBUG_ERROR("ArduinoIoTCloudTCP::%s MQTT client connection lost", __FUNCTION__);
+  } else {
+    _mqttClient.unsubscribe(_shadowTopicIn);
+    _mqttClient.unsubscribe(_dataTopicIn);
+    /* TODO add device topic */
+    _mqttClient.stop();
+  }
+  DEBUG_INFO("Disconnected from Arduino IoT Cloud");
```

Comment:
`Disconnect` state is triggered also in case we hit the limit of last values requests to the cloud. In this case the happy path will print only

```
... something about last values requests ...
Disconnected from Arduino IoT Cloud
```

If during a "scheduled disconnection" the client loses the connection it will print both

```
... something abot last values request ...
ArduinoIoTCloudTCP::handle_Disconnect MQTT client connection lost
Disconnected from Arduino IoT Cloud
```

I think the two prints have different meanings and thus the different severity, but the final outcome is the same for both paths: The board gets disconnected from the cloud.